aicsimageio/readers/tiff_reader.py

```diff
@@ -175,53 +193,96 @@ def _create_dask_array(self) -> da.Array:
         image_data: da.Array
             The fully constructed and fully delayed image as a Dask Array object.
         """
+        # Always add the plane dimensions if not present already
+        if DimensionNames.SpatialY not in self.chunk_by_dims:
+            self.chunk_by_dims.append(DimensionNames.SpatialY)
+        if DimensionNames.SpatialX not in self.chunk_by_dims:
+            self.chunk_by_dims.append(DimensionNames.SpatialX)
+        if DimensionNames.Samples not in self.chunk_by_dims:
+            self.chunk_by_dims.append(DimensionNames.Samples)
```

Comment:
This might be better in a smaller for loop
```suggestion
        for dim in (DimensionNames.SpatialY, DimensionNames.SpatialX, DimensionNames.Samples):
            if dim not in self.chunk_by_dims:
                self.chunk_by_dims.append(dim)
```

Reply:
This is in general a good suggestion as I expect this will come in other readers so I will probably make this a constant in the `dimensions.py`.

```python
for dim in dimensions.REQUIRED_CHUNK_BY_DIMS:
```